### ISSUE TITLE
Improve badges title and make them more visual

### ISSUE THREAD

**Issue by darechan** [unstructuredstudio/zubhub#642]
# Feature Request

## 1. Designing badges for achievements 

Zubhub currently have a variety of `Badges` for creators on the platform as follows (Ascending)
```
1. Hatchling
2. Flying Bird
3. Master of the sky
4. Expert Builder
5. Helping Hand
6. Always Available
7. Expert Advisor
8. Getting Famous
9. Person of Interest
10. Popular Projects
11. Idea Factory
12. Interesting Projects
13. Favourite Kid
14. Captain Projects
```
But these badges currently are only in a form of text
![image](https://user-images.githubusercontent.com/93364240/224487895-f6cbc86b-5185-429a-af36-b76396362418.png)

Having visual badges that could be displayed on the profile and downloadable/sharable for the user would look a lot more pleasing.

Also if we could add in a few rewards like a custom frame unlocking or a special feature on earning a badge then that would make the website more interactive! Would love to have UI and UX assistance for the same!

**Comment by Rosietta-UIUX** [comment]
Wow @darechan I just opened an issue similar to this (Badge redesign) how can we merge it? I love the idea

**Comment by darechan** [comment]
@Rosietta-UIUX Please attach the idea here and add a comment on the other issue with a comment `merging with #642` and press the `close with comment` button

**Comment by Rosietta-UIUX** [comment]
@darechan Alright let me do that

**Comment by darechan** [comment]
@Rosietta-UIUX Can you upload the proposed designs here?

**Comment by Rosietta-UIUX** [comment]
Hello @sethiastha, @srish, @tuxology

While browsing the Zubhub website with my younger sister, she pointed out that the badges on the site are plain and unattractive, resembling tags rather than fun badges. Therefore, I would like to request to be assigned to this issue along with @darechan so that we can redesign the badges and make them more visually appealing.

**Current Badges**
![Polish_20230311_153020559](https://user-images.githubusercontent.com/123890450/224491179-6fdf237b-c057-4a3c-b54b-e73ebf6a3bf8.jpg)

**Proposed Redesign** 
![Polish_20230311_153045734](https://user-images.githubusercontent.com/123890450/224491211-8212ea05-ee9f-488f-a837-36b4865fb2dd.jpg)

**Comment by kan-ma** [comment]
Proposed badges designs
<img width="200" alt="hatchling" src="https://user-images.githubusercontent.com/35083425/224573565-5af3cbbb-f940-4d42-8efb-d56249efe34c.png">
<img width="200" alt="flying bird" src="https://user-images.githubusercontent.com/35083425/224573566-3112d549-91fa-4f50-8baf-b7b5cf1f57a9.png">

**Comment by kan-ma** [comment]
badges design
<img width="200" alt="hatchling" src="https://user-images.githubusercontent.com/35083425/224574799-b4fcbc78-54a6-4b90-a33e-d3333f334e35.png"> hatchling
<img width="200" alt="flying bird" src="https://user-images.githubusercontent.com/35083425/224574806-098ba43e-1260-4567-bbee-d3b040308c8f.png"> flying bird

**Comment by tuxology** [comment]
While you all think about it, please ensure that images should be open access/licensed under a creative commons or should be self drawn. We cannot use random birds and emojis from the internet. Look at https://openclipart.org for a starting point.

**Comment by Rosietta-UIUX** [comment]
Thank you @tuxology

**Comment by ferindesigns** [comment]
@darechan What do you think?
This
![Blue Red Colorful Good Job Student Stickers](https://user-images.githubusercontent.com/108564127/225315976-00ebd3ba-78fb-483c-a2f5-1ea8e1da8c0a.gif)
or this
![Blue Red Colorful Good Job Student Stickers (2)](https://user-images.githubusercontent.com/108564127/225316168-ade12584-a1af-4602-8b60-4d84536d5835.png)

**Comment by darechan** [comment]
> @darechan What do you think?
> This
> ![Blue Red Colorful Good Job Student Stickers](https://user-images.githubusercontent.com/108564127/225315976-00ebd3ba-78fb-483c-a2f5-1ea8e1da8c0a.gif)
> or this
> ![Blue Red Colorful Good Job Student Stickers (2)](https://user-images.githubusercontent.com/108564127/225316168-ade12584-a1af-4602-8b60-4d84536d5835.png)
> 

I think this badge is good but the doesn't matches the theme of the website don't you think so too?

**Comment by ferindesigns** [comment]
> > @darechan What do you think?
> > This
> > ![Blue Red Colorful Good Job Student Stickers](https://user-images.githubusercontent.com/108564127/225315976-00ebd3ba-78fb-483c-a2f5-1ea8e1da8c0a.gif)
> >       
> >         [
> >           
> >         
> >             ![Blue Red Colorful Good Job Student Stickers](https://user-images.githubusercontent.com/108564127/225315976-00ebd3ba-78fb-483c-a2f5-1ea8e1da8c0a.gif)
> >           ](https://user-images.githubusercontent.com/108564127/225315976-00ebd3ba-78fb-483c-a2f5-1ea8e1da8c0a.gif)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://user-images.githubusercontent.com/108564127/225315976-00ebd3ba-78fb-483c-a2f5-1ea8e1da8c0a.gif)
> >         
> >       
> > or this
> > ![Blue Red Colorful Good Job Student Stickers (2)](https://user-images.githubusercontent.com/108564127/225316168-ade12584-a1af-4602-8b60-4d84536d5835.png)
> 
> I think this badge is good but the doesn't matches the theme of the website don't you think so too?

okay will work on something else.

**Comment by comfy96** [comment]
what do you think about this? @darechan 
![Group 69 (2)](https://user-images.githubusercontent.com/57982384/225345875-08053dc1-b6e9-4ea9-b935-9f406ece035c.png)

I'm still working on some, just wanted to show you this first.

**Comment by comfy96** [comment]
a smaller one @darechan 
![helping hand 2](https://user-images.githubusercontent.com/57982384/225346521-f999f7c9-7429-4530-949e-dd4d199217a3.png)

**Comment by darechan** [comment]
@comfy96 the may be you could use some graffiti fonts and make it more playful as our intended audience is young children

**Comment by comfy96** [comment]
okay, I'd work on something right away. @darechan

**Comment by Ami112-sketch** [comment]
Hi @tuxology
It will be a nice feature to add a brief description of each badge( what each badge means or entails) under each of the visuals and use a graphic image/ visual badge that gives  meaning to the children( we are providing them with options i.e meaning via the image and also the brief description beneath the visual
![artworkkkk](https://user-images.githubusercontent.com/105370797/225372916-ea885a6a-062d-405e-b3b7-98467ed41aeb.jpg)

**Comment by Ami112-sketch** [comment]
> Hi @tuxology It will be a nice feature to add a brief description of each badge( what each badge means or entails) under each of the visuals and use a graphic image/ visual badge that gives meaning to the children( we are providing them with options i.e meaning via the image and also the brief description beneath the visual ![artworkkkk](https://user-images.githubusercontent.com/105370797/225372916-ea885a6a-062d-405e-b3b7-98467ed41aeb.jpg)

@darechan

**Comment by comfy96** [comment]
what do you think @darechan 
![FAVORITE](https://user-images.githubusercontent.com/57982384/225373611-80d91fd3-1657-48ad-955e-456d4a0fc8de.png)


favorite kid: looking all cheerful

**Comment by ferindesigns** [comment]
@darechan what do you think about this for idea factory?
![Colorful Lightbulb Switched On Sticker](https://user-images.githubusercontent.com/108564127/225374564-1daae1c9-79c2-448d-a296-d006c89c2391.png)

**Comment by darechan** [comment]
@ferindesigns Neat work!

**Comment by ferindesigns** [comment]
> @ferindesigns Neat work!

Thank you

**Comment by ferindesigns** [comment]
![Colorful Lightbulb Switched On Sticker (2)](https://user-images.githubusercontent.com/108564127/225441608-25a264c9-d7ee-47df-969b-6a69e7c53de5.png)
![Colorful Lightbulb Switched On Sticker (4)](https://user-images.githubusercontent.com/108564127/225441622-d393440f-4b82-4075-bc03-3574d031740f.png)
@darechan what do you think?

**Comment by comfy96** [comment]
![badges 3](https://user-images.githubusercontent.com/57982384/225814898-6bd78624-6f55-40f0-86d3-6e745deae869.png)

@darechan @tuxology @srish @AsthaSethi

**Comment by robotdiner** [comment]
Hi there! I've made art for of the badges, I've had a lot of fun making these:

1. **Captain Projects**
![Captain_Projects](https://user-images.githubusercontent.com/64199933/226356223-ee7bb9db-b762-4444-be40-abb3f686e4f8.png)

2. Expert Builder
![Expert_Builder](https://user-images.githubusercontent.com/64199933/226356230-5729205f-a9f2-465f-8ec6-66ca6b391181.png)

3. Interesting Projects
![Interesting_Projects](https://user-images.githubusercontent.com/64199933/226356200-982513f7-7330-4ac9-912f-395f5f6cf01f.png)

4. Flying Bird
![Flying_Bird](https://user-images.githubusercontent.com/64199933/226356241-481278b7-7108-4e22-a65d-e62e6da327a9.png)

5. Hatchling
![Hatchling](https://user-images.githubusercontent.com/64199933/226356243-5b893986-4922-4a29-8809-caf2bd827721.png)

6. Idea Factory
![Idea_Factory](https://user-images.githubusercontent.com/64199933/226356252-6ec85cf9-561d-4176-a311-b48a51836c28.png)

For the art style, I choose a handmade, friendly theme with many bright colors, and at large made art that was distinct for each achievement, something the creators would be excited to have. For the "Interesting projects" art, I used a baby looking curiously because I think it would be sweet to have some funny and expressive art. I kept a theme of tinkering and exploration across the art. If you have any feedback, I'd love to hear it. Thank you!

@darechan @tuxology

**Comment by mehreeee** [comment]
@srish @tuxology  Can I have this task assigned to me?

**Comment by DonPresh** [comment]
Hi, @srish @tuxology I am an outreachy applicant trying to pick up this task. Is there an existing illustration system for Zubhub that I should follow or do I go ahead and design the badges?

**Comment by Shulamite54** [comment]
@srish @tuxology good day.
Please can I be assigned to this task.
Thanks

**Comment by JulietMadu** [comment]
@tuxology I can work on this

**Comment by ekdumswast** [comment]
Hi @srish @tuxology I am looking for projects to contribute to in design, I find this project a good first issue to work on so do we get it assigned or can we start contributing our designs here?

**Comment by Marhiposa** [comment]
HI @tuxology @srish @comfy96 I see that this issue is still opened pls can you assign it to me so i improve the badges title and make it more visual?

**Comment by JessicaItepu** [comment]
Hello @srish @tuxology, I would love to be assigned to this project, i have a few ideas i could implement. Thank you.

**Comment by DonPresh** [comment]
I went through the website and saw that this issue has not been resolved so I picked a few assets from the website to create an illustration style for the badge design. 

 I will like to know if this matches the theme of the website and would appreciate feedback so I can go ahead and complete the remaining badges @srish @tuxology @darechan 

![IMG_0384](https://github.com/unstructuredstudio/zubhub/assets/100778015/9792861e-35e3-479b-83ca-d3ac1cec5ffc)

P.S. I can also attach the .svg files as they are all self drawn

**Comment by Marhiposa** [comment]
@comfy96 @darechan what do you think about this designs?

![Frame 1](https://github.com/unstructuredstudio/zubhub/assets/146780247/79429772-757b-4627-859d-059ddefa2847)

still designing but i would like to know if you like the theme or is there anything you would like me to change ? @tuxology i got some of the images from openclipart.

**Comment by Boluwatifey1** [comment]
@comfy96 @DonPresh please where do i get the list of the badges

**Comment by Boluwatifey1** [comment]
@comfy96 @DonPresh please where do i get the list of the badges

**Comment by DonPresh** [comment]
> @comfy96 @DonPresh please where do i get the list of the badges


You can find them on the first comment in this pr

**Comment by DonPresh** [comment]
> @comfy96 @DonPresh please where do i get the list of the HI, Pls scroll to the top of the this thread to see the list

**Comment by DonPresh** [comment]
> # Feature Request
> ## 1. Designing badges for achievements
> Zubhub currently have a variety of `Badges` for creators on the platform as follows (Ascending)
> 
> ```
> 1. Hatchling
> 2. Flying Bird
> 3. Master of the sky
> 4. Expert Builder
> 5. Helping Hand
> 6. Always Available
> 7. Expert Advisor
> 8. Getting Famous
> 9. Person of Interest
> 10. Popular Projects
> 11. Idea Factory
> 12. Interesting Projects
> 13. Favourite Kid
> 14. Captain Projects
> ```
> 
> But these badges currently are only in a form of text ![image](https://user-images.githubusercontent.com/93364240/224487895-f6cbc86b-5185-429a-af36-b76396362418.png)
> 
> Having visual badges that could be displayed on the profile and downloadable/sharable for the user would look a lot more pleasing.
> 
> Also if we could add in a few rewards like a custom frame unlocking or a special feature on earning a badge then that would make the website more interactive! Would love to have UI and UX assistance for the same!

@Boluwatifey1 here

**Comment by Marhiposa** [comment]
@DonPresh  can we work together to get great designs ? i dont mind collaborating. let me know what you think.

**Comment by DonPresh** [comment]
> @DonPresh can we work together to get great designs ? i dont mind collaborating. let me know what you think.

Yes, absolutely. 
sorry for the late response

**Comment by ladyami** [comment]
As a designer, my goal is to create badges for Zubhub creators that are not only visually appealing but also informative and meaningful. To achieve this goal, I first understood the purpose of each badge. The Flying Bird and Master of the Sky badges represent a creator's growth and development. The Expert Builder and Helping Hand badges represent a creator's ability to build high-quality content and help others. The Always Available and Expert Advisor badges represent a creator's positive attitude, willingness to engage with their users, and expertise in their field. The Getting Famous, Person of Interest, Popular Projects, Idea Factory, and Interesting Projects badges represent a creator's growing popularity and the unique and engaging content that they create. The Favourite Kid badge represents the creator's popularity with their users, while the Captain Projects badge represents the creator's leadership and ability to manage their projects.

After understanding the purpose of each badge, I brainstormed ideas for icons, colors, and text that would best represent each badge. I chose blue and yellow as the main colors for the badges, as they are both associated with the sky and flight, which are symbolic of growth and potential. I used a bird icon for the Flying Bird badge to represent a creator's growth and development. For the Master of the Sky badge, I chose a bird icon soaring through the sky and added black to represent luxury and excellence. which are both qualities of a master of their craft.

I applied a similar approach when designing the other badges, carefully considering the icons, colors, and text for each badge to create a design that is visually appealing and meaningful. I made sure that the badges were consistent with the overall branding of the Zubhub platform by using the same colors and fonts that are used throughout the platform. This helps to create a sense of unity and consistency for users.

In conclusion, I hope that the badges I designed we be both visually appealing and meaningful to Zubhub creators. I hope that these badges will encourage creators to continue contributing to the platform and creating high-quality content.

[https://www.figma.com/file/CSYODN3L9Vp5BsokH611HI/zubhub-badges?type=design&node-id=2%3A480&mode=design&t=Hln4mXCMCDymNsdy-1](url)

![MacBook Air - 1](https://github.com/unstructuredstudio/zubhub/assets/99187115/c43261f9-69cd-4463-822b-ba756c3b4ae4)

@srish Could you please review the badge proposal Your feedback and suggestions would be much appreciated.

**Comment by hdtouch101** [comment]
Hi, can I proceed with solving this issue?

**Comment by srish** [comment]
@DonPresh, it looks like your designs are in line with the website's theme, which is great. Would you be able to provide a few additional variations and maybe suggest alternative names for badges? We've been considering renaming some of them, as a few might be unclear to high school students.

@sethiastha, @Faridah-ux, could you please share your thoughts and ideas on this matter as well? Your input would be valuable.

**Comment by DonPresh** [comment]
@srish Thank you for the feedback. Yes, I can provide more variations for the badges.
About the renaming, which of the badges are you considering renaming? Maybe I can come up with some cool suggestions. 

Personally, I think I like Captain Projects a lot. Sounds like a Disney or  Nickelodeon character.

**Comment by productguyyy** [comment]
@srish I will love for you to see the ideals I have of the badges, please can you assign the task to me?

**Comment by N-TITIAN** [comment]
@srish i will love to work on the badges or even collaborate with another designer , can i start right away?

**Comment by outcast-02** [comment]
Hey @DonPresh I would like to collab with you if you are okay with it! Let me know and we can connect and share some ideas ^^

**Comment by pachougu** [comment]
@DonPresh can we collaborate on this i also have a good ideal that can march yours

**Comment by pachougu** [comment]
I love us to work on this

**Comment by Temabo** [comment]
@DonPresh Soild work! The badge designs look good, I have some design ideas and would love to collaborate with you on figma.

@srish here are some suggestions for alternative badge titles;
Newbie Explorer
Soaring High
Sky Commander
Skillful Creator
Friendly Guide
Ever-Present Buddy
Top Advice Guru
Rising Star
Community Influencer
Hot Picks
Creative Spark
Intriguing Ventures
Student Favorite
Project Captain

What do you think about these titles?

**Comment by outcast-02** [comment]
Hi @srish I have a few suggestions too for the title of the badge if you like it we can proceed with the design that will suit the term.
1.Discovery Dynamo
2.Tech Wizard
3.Team Spirit Trooper
4.Knowledge Ninja

**Comment by DonPresh** [comment]
Hey @outcast-02 @Temabo @pachougu

Sure, we can collaborate. Maybe bounce off ideas here so anyone else interested can join us. 
Does that work?

**Comment by outcast-02** [comment]
@DonPresh  Sure I am up!  let me know if we can connect on zulip chat

**Comment by mehreeee** [comment]
@srish I have a few suggestions for the names
Hatchling ~ Novice
Flying Bird ~ Amateur
Master of the sky ~ Senior
Expert Builder ~ Enthusiast
Helping Hand ~ Professional
Always Available ~ Expert
Expert Advisor ~ Leader
Getting Famous ~ Architect
Person of Interest ~ Veteran
Popular Projects ~ Achiever
Idea Factory ~ Master Achiever
Interesting Projects ~ Super Commando
Favorite kid ~ Grandmaster
Captain project ~ Prodigy

**Comment by mehreeee** [comment]
I think we should keep the names fun and simple for high school kids

**Comment by mehreeee** [comment]
> Hey @outcast-02 @Temabo @pachougu
> 
> Sure, we can collaborate. Maybe bounce off ideas here so anyone else interested can join us. Does that work?

I'm interested

**Comment by DonPresh** [comment]
Hi @srish I was thinking for renaming the badges, we could explore the idea of creating a more engaging and thematic title ranking system for our reward badges. 

I think having a themed approach could make the experience even more exciting for our users since they are young and impressionable. 

For instance, we could consider themes like a military theme, something magical or wizard-related, or even something fun like poultry or gaming.
As an example, within the Poultry theme, we could have badges named after various aspects of poultry, such as "Hatchling," "Rooster," "Nest Builder," and so on. This approach could not only make the badges more interesting but also add an extra layer of fun and creativity to the platform. 

What are your thoughts on this?

**Comment by Faridah-ux** [comment]
> @DonPresh, it looks like your designs are in line with the website's theme, which is great. Would you be able to provide a few additional variations and maybe suggest alternative names for badges? We've been considering renaming some of them, as a few might be unclear to high school students.
> 
> @sethiastha, @Faridah-ux, could you please share your thoughts and ideas on this matter as well? Your input would be valuable.


> @DonPresh, it looks like your designs are in line with the website's theme, which is great. Would you be able to provide a few additional variations and maybe suggest alternative names for badges? We've been considering renaming some of them, as a few might be unclear to high school students.
> 
> @sethiastha, @Faridah-ux, could you please share your thoughts and ideas on this matter as well? Your input would be valuable.

Hello @srish In my opinion, I think 14 badges is a lot. I think it should be reduced to 7 badges. This is because we don't have enough features to mark these milestones.

We could select 7 badge names from the ones we have already.

Like **Hatchling**- For when they create an account.
**Flying Bird**- For when they successfully create a project.
**Master of the sky**- For when they have created 3 projects.
**Helping Hand**- For when they create a Team.
**Getting Famous-** For when they get 50 views on their project.
**Person of Interest**- For when they get 20 claps on their project.
**Captain Projects**- For when they have created 10 or more projects.


Nice work @DonPresh

**Comment by pachougu** [comment]
I agree to what @ferindesigns said, I also explore so designs for the badge using 3d effect to make more fun and the i also added it to the dashboard to see the look and feel of it.  @srish I believe if we create fun badge and make this product more fun to our end user with the badges it will attract and gain the mid of users, just like togo app did to it users then cause everyone wants to scale up to profession and that made togo app more fun among peers. 
![drafts - edit drafts - top](https://github.com/unstructuredstudio/zubhub/assets/146971182/97bc8dd8-f795-4226-9b48-112acb2aa0e2)

![Frame 9022](https://github.com/unstructuredstudio/zubhub/assets/146971182/3fb6ca48-9520-40de-80f6-edebba178acc)

**Comment by N-TITIAN** [comment]
> @srish i will love to work on the badges or even collaborate with another designer , can i start right away?

while waiting for the new badge names i decided to design a badge for "Idea Factory" badge
![Frame 2](https://github.com/unstructuredstudio/zubhub/assets/81820848/cfc92fb4-d88c-441d-89e0-84f22475d92b)

**Comment by outcast-02** [comment]
Hi @srish I looked into the badge improvement and came up with an idea.  
the idea is to focus on the badge shape first then the icon/vector. I know this might seem like extra work to focus on but I did some research and realized that we should give the users some extra options. like the color of the badge and the icon/vector inside it.. after all, getting the same-looking achievement batch with others might not be that exciting when they worked hard to reach a milestone to get the achievement badge. So I made  one batch of designs sharing that here
![badge](https://github.com/unstructuredstudio/zubhub/assets/75933388/8805289d-c00a-4f1b-b1b5-06a6601b1eff)

**Comment by Mbeute23** [comment]
@DonPresh Pls can i going the team to work on the badges together?

**Comment by Shulamite54** [comment]
@srish , Here are my few suggestion.

Flying Bird - Friendly Feathered Friend 
Master of the Sky - Sky's Superhero 
Helping Hand - Helping Buddy
Getting Famous - Rising Star
Person of Interest - Cool Character
Captain Project - Captain Adventure

**Comment by DonPresh** [comment]
> @srish Thank you for the feedback. Yes, I can provide more variations for the badges. About the renaming, which of the badges are you considering renaming? Maybe I can come up with some cool suggestions.
> 
> Personally, I think I like Captain Projects a lot. Sounds like a Disney or Nickelodeon character.

@srish  I illustrated two more variations for the badges. I’ve not drawn more because I’m holding off till decisions are made on renaming the badges 

![IMG_0406](https://github.com/unstructuredstudio/zubhub/assets/100778015/0f38aa2d-be51-4b68-bed5-78a71670241e)
![IMG_0407](https://github.com/unstructuredstudio/zubhub/assets/100778015/d4b88c1e-3a76-4614-b596-656d3799d868)

**Comment by Mbeute23** [comment]
Issue: Improve badges tittle and make them more visual

Proposed solution:

My first step was to look for vector illustrations for each badge name (vector image illustrating each badge) then look for a badge that i will go with.

I choose vector image because it's user friendly and will direct the user to what type of badge it is visually.

Also, going to the color i choose the Zubhub color through out the visuals 

Here is the link to the figma file: https://www.figma.com/file/ByALA1tM5mTZCfLz5Kr3Ko/Outreachy-Zubhub-Design-(Contribution-to-issues)?type=design&node-id=315%3A697&mode=design&t=4UgRB1HUe5y2rx5J-1

![Uploading ISSUE_ Improve badges title and make them more visual _642.png…]()


@srish what do you think about this designs?

**Comment by Mbeute23** [comment]
Issue: Improve badges tittle and make them more visual

Proposed solution:

My first step was to look for vector illustrations for each badge name (vector image illustrating each badge) then look for a badge that i will go with.

I choose vector image because it's user friendly and will direct the user to what type of badge it is visually.

Also, going to the color i choose the Zubhub color through out the visuals 

Here is the link to the figma file: https://www.figma.com/file/ByALA1tM5mTZCfLz5Kr3Ko/Outreachy-Zubhub-Design-(Contribution-to-issues)?type=design&node-id=315%3A697&mode=design&t=4UgRB1HUe5y2rx5J-1

![Uploading ISSUE_ Improve badges title and make them more visual _642.png…]()


@srish what do you think about this designs?

**Comment by Mbeute23** [comment]
Issue: Improve badges tittle and make them more visual

Proposed solution:

My first step was to look for vector illustrations for each badge name (vector image illustrating each badge) then look for a badge that i will go with.

I choose vector image because it's user friendly and will direct the user to what type of badge it is visually.

Also, going to the color i choose the Zubhub color through out the visuals 

Here is the link to the figma file: https://www.figma.com/file/ByALA1tM5mTZCfLz5Kr3Ko/Outreachy-Zubhub-Design-(Contribution-to-issues)?type=design&node-id=315%3A697&mode=design&t=4UgRB1HUe5y2rx5J-1

<img width="759" alt="Screen Shot 2023-10-07 at 6 59 57 PM" src="https://github.com/unstructuredstudio/zubhub/assets/146943654/c8f38bfa-73b6-4b2d-8379-2609a1d72f3f">


@srish what do you think about this designs?

**Comment by OnolexTim** [comment]
@DonPresh I'd like to collaborate with you on this design. Do you mind having more than one collaborator?

**Comment by GloriaKaduru** [comment]
Hello! Working with what @DonPresh illustrated and the new badge titles and descriptions @Faridah-ux made, I worked on improving the overall user experience of the badges feature.

1. **Profile Page with new Illustrated Badges:** Here, I tweaked @DonPresh's illustrations a little bit and added the badges to the Badge section. I would love to collaborate with her to flesh it out fully and craft out more illustrations for the badges if this is greenlighted.

<img width="390" alt="New Badges" src="https://github.com/unstructuredstudio/zubhub/assets/82684587/0343185b-4147-4b9d-85a3-ab94aa0ad8b3">



2. **Getting a new Badge:** I noticed that in the current flow for getting badges, there is no confirmation/announcement to the user that they have gotten a badge and why. Here, I created a modal that pops up whenever a milestone is ahcieved. The modal tells the user that they have achieved a badge and why and also prompts them to share it. Here's a modal for the **_Hatchling_** Badge. 

<img width="390" alt="Getting a new Badge" src="https://github.com/unstructuredstudio/zubhub/assets/82684587/bb22f63f-1271-4bad-9a96-c75b18b15cec">



2.1.**Modal for Flying Bird Badge (without the confetti)**
<img width="294" alt="Flying Bird Modal" src="https://github.com/unstructuredstudio/zubhub/assets/82684587/1b0fc4c8-341b-4d4e-bee8-1741d5a9783d">



3. **Badge with Badge Info:** Here, I added a mini-description of the badge in form of a tooltip. This occurs when the user taps on the badge (on mobile) or click/hovers on it (on desktop). This is helpful in cases where the user has forgotten why they got the badge in the first place or they maybe want to show off! :)


<img width="390" alt="New Badges (Badge Info)" src="https://github.com/unstructuredstudio/zubhub/assets/82684587/bcab610c-44b0-4a10-ba6d-570814b1058b">




Please let me know what your thoughts are on this. @Srish @tuxology . Thank you!

**Comment by DonPresh** [comment]
@GloriaKaduru I love how you used the badges on this design. Pending when @srish gives approval we can work on this together.

**Comment by aqsaaqeel** [comment]
love the badges design @DonPresh and how you have incorporated the badges on the profile. Great work!!
